keda/values.yaml

```diff
@@ -35,6 +35,24 @@ image:
 # Kubernetes cluster domain
 clusterDomain: cluster.local
 
+vpa:
```

Comment:
Can you also add the comment stuff I force edited into keda app ?

Reply:
The one here https://github.com/giantswarm/keda-app/commit/a4e51a8a4518654e7f4f4f14811fcec6701bdc05

Reply:
Done